packages/dev/core/src/Engines/Extensions/engine.multiview.ts

```diff
@@ -90,6 +106,50 @@ Engine.prototype.bindMultiviewFramebuffer = function (_multiviewTexture: RenderT
     }
 };
 
+Engine.prototype.createSpaceWarpRenderTargetTexture = function (width: number, height: number, motionVectorTexture: WebGLTexture, depthStencilTexture: WebGLTexture) {

```

Comment:
What is special about this ? or different from a regular RTT in multiview ?

Reply:
The function was different in the sense that the WebGLTextures were already defined. I have moved everything to a single function, reusing as much code as possible.